types/nodecredstash/nodecredstash-tests.ts

```diff
@@ -0,0 +1,59 @@
+import Credstash = require('nodecredstash');
+
+const credstash = Credstash({
```

Comment:
This doesn't match the usage example on npm. It looks like `Credstash` is a class. Is this intentional?

https://www.npmjs.com/package/nodecredstash

```js
let Credstash = require('nodecredstash');
 
let credstash = new Credstash({table: 'credential-store', awsOpts: {region: 'us-west-2'}});
 
credstash.putSecret({name: 'Death Star vulnerability', secret: 'Exhaust vent', version: 1, context: {rebel: 'true'}})
  .then(() => credstash.getSecret({name: 'Death Star vulnerability', version: 1, context: {rebel: 'true'}})
  .then(secret => console.log(secret));
```

Reply:
Yes - I originally exposed it as a class, but in fact the source shows that the module exports a function that returns a new instance of an internal `Credstash` class - https://github.com/DavidTanner/nodecredstash/blob/master/js/index.js#L15. I would say that the example in the package README is actually misleading